package.json

```diff
@@ -39,6 +39,7 @@
     "@wordpress/i18n": "^4.2.1",
     "@wordpress/icons": "^5.0.1",
     "@wordpress/jest-preset-default": "^7.1.0",
+    "@wordpress/notices": "^1.12.0",
```

Comment:
Installing this development package because of [this import of @wordpress/notices](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3265/files#diff-249e814fd2610cf7f4a42eda57b09ca90b15c9c0176ad12bc38931da3e0bfaa8R3). Eslint displays an "'@wordpress/notices' should be listed in the project's dependencies." problem otherwise.

I installed an [old version](https://github.com/WordPress/gutenberg/blob/ddadd1a95d18270908ac4a1fd8d6e354cfadf61c/packages/notices/CHANGELOG.md), which is  the same one we had in our modules' dependencies, because newer versions of the package triggered errors when running the JS unit tests.

An alternative would have been using the already installed `@woocommerce/data` package. Happy to update it if it's more convenient.